src/components/Speciality/Projects.js

```diff
@@ -11,31 +11,38 @@ const Emphasis = styled.em`
   color: ${props => props.theme.colors.secondaryText};
 `
 
-const PosterLinks = ({ project }) => (
-  <AnimatedLink to={`/case-study/${project.slug}`} title={project.title}>
-    <section
-      style={{
-        background: `#${project.posterColor}`
-      }}
-    >
-      <CardHeader>
-        <CardTitle reverse noPadding bigger>
-          {project.title}
-        </CardTitle>
-        <BodyPrimary reverse muted>
-          {project.introSentence.introSentence}
-        </BodyPrimary>
-      </CardHeader>
-      <PosterImage justifyCenter alignCenter color={project.posterColor}>
-        <img
-          alt={project.posterImage.title}
-          src={project.posterImage.file.url}
-          style={{ maxHeight: '100%' }}
-        />
-      </PosterImage>
-    </section>
-  </AnimatedLink>
-)
+const PosterLinks = ({ project }) => {
+  const backgroundIsDark = project.posterColor === 'ff4e5b' // Learnably poster color
```

Comment:
I would rename to `isDarkBackground`. It's better to have a boolean variable start with "is".
By the way is this working? In theory it looks good but when I go to `http://localhost:8000/speciality/data-analysis/` the text is still dark. Is this my local environment?